docs/troubleshooting.md

```diff
@@ -59,4 +59,10 @@ ssh-keygen \
   /path/to/key.pub
 ```
 
-You can then use the signed SSH key to SSH via `ssh -i /path/to/key.pub user@server`. 
\ No newline at end of file
+You can then use the signed SSH key to SSH via `ssh -i /path/to/key.pub user@server`. 
+
+## Using kssh with jumpboxes and bastion hosts
+
+kssh does work correctly with jumpboxes and bastion hosts. Eg the command `kssh -J jumpbox.example.com server.internal`
```

Comment:
this is a weirdly worded sentence for a troubleshooting doc. "oh you're having issues? well, it _does_ work correctly."
suggest moving this to the README with a little more of a "how-to" vibe.

Reply:
Done